tests/test_builtin_casters.py

```diff
@@ -115,13 +115,12 @@ def test_bytes_to_string():
     """Tests the ability to pass bytes to C++ string-accepting functions.  Note that this is
     one-way: the only way to return bytes to Python is via the pybind11::bytes class."""
     # Issue #816
-    import sys
-    byte = bytes if sys.version_info[0] < 3 else str
+    bytes_ = bytes if pytest.PY2 else str
```

Comment:
Are you sure this is needed? See: https://six.readthedocs.io/#six.binary_type

> Python 2.6 and 2.7 include bytes as a builtin alias of str, so six’s version is only necessary for Python 2.5 compatibility.

Reply:
Lemme check. FWIW I just converted existing code, didn't think too deeply about it.

Reply:
Done. The logic in the function seemed wrong - it didn't really test `bytes` in Python 3.

Reply:
Pinging @rwgk, since it's a test involving str/bytes ;-) This can just be merged, though; looks OK to me.